jinja2/base.html

```diff
@@ -17,6 +17,10 @@
         "noindex, nofollow"
     {%- endif -%}
   >
+
+  <!-- because this is non-blocking, preload as early as possible -->
```

Comment:
This HTML comment seems more appropriate in ``preload.html``, since it goes with the markup there.  If you just want a comment for developers, I'd suggest a ``{# jinja comment #}``.

Reply:
Makes sense.